.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+version: 2.1
+orbs:
+  slack: circleci/slack@3.4.2
+jobs:
+  generate docs:
+    docker:
+      - image: cimg/node:lts
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
+    resource_class: xlarge     
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: npm install
+          command: sudo npm install -g lerna && npm install
```

Comment:
@meenahoda 

I would rename this job "install lerna" and change the command to just `sudo npm install -g lerna`